waspc/src/Wasp/Generator/WebAppGenerator/Common.hs

```diff
@@ -48,15 +51,29 @@ webAppRootDirInProjectRootDir = [reldir|web-app|]
 webAppSrcDirInWebAppRootDir :: Path' (Rel WebAppRootDir) (Dir WebAppSrcDir)
 webAppSrcDirInWebAppRootDir = [reldir|src|]
 
+-- | Path to generated web app src/ directory, relative to the root directory of the whole generated project.
 webAppSrcDirInProjectRootDir :: Path' (Rel ProjectRootDir) (Dir WebAppSrcDir)
 webAppSrcDirInProjectRootDir = webAppRootDirInProjectRootDir </> webAppSrcDirInWebAppRootDir
 
--- * Templates
-
 -- | Path in templates directory where web app templates reside.
 webAppTemplatesDirInTemplatesDir :: Path' (Rel TemplatesDir) (Dir WebAppTemplatesDir)
 webAppTemplatesDirInTemplatesDir = [reldir|react-app|]
 
+-- | Path to the web app templates src/ directory, relative to the web app templates directory.
+srcDirInWebAppTemplatesDir :: Path' (Rel WebAppTemplatesDir) (Dir WebAppTemplatesSrcDir)
+srcDirInWebAppTemplatesDir = [reldir|src|]
+
+-- * Templates
```

Comment:
Why did this title get moved? I think it was supposed to capture the Templates Paths above also.

Reply:
We're removing all titles